packages/flutter/lib/src/services/image_stream.dart

```diff
@@ -154,11 +154,14 @@ class ImageStreamCompleter {
   /// Adds a listener callback that is called whenever a concrete [ImageInfo]
   /// object is available. If a concrete image is already available, this object
   /// will call the listener synchronously.
+  /// The listener will be passed a flag indicating whether a synchronous call
```

Comment:
add a blank line above the new paragraph so that it does an actual paragraph.

also, super trivial nit: only one space after periods in dartdocs.